src/_blog/rust-libp2p-052-release.md

```diff
@@ -0,0 +1,233 @@
+---
+tags:
+- libp2p
+- rust
+title: rust-libp2p v0.52 release spotlight
+description: Walking through the rust-libp2p v0.52 release
+date: 2023-06-19
+permalink: "/2023-06-19-rust-libp2p-052"
+translationKey: ''
+header_image: /rust-libp2p-2022-header.png
+author: Thomas Eizinger
+---
+
+# rust-libp2p v0.52 release spotlight
+
+With the `v0.52` release, the `rust-libp2p` maintainer team tried something new.
+Breaking changes - even in the presence of an obvious or easy upgrade path - are a burden for our users.
+They easily trigger a cascade of breaking changes for downstream users.
+
+Coming up to the `v0.52` release, we consciously held back and batched up breaking changes and released several features as patch releases instead.
+As a result, `v0.52` is unfortunately quite large.
+This blogpost aims to highlight some of the more exciting changes:
```

Comment:
```suggestion
This blogpost highlights the most exciting changes:
```

Not a native speaker. To me "the more exciting changes" sounds like the other non-highlighted changes are bad.